Sources/PT.PM.JavaParseTreeUst/Converter/StatementVisitor.cs

```diff
@@ -62,6 +63,23 @@ public UstNode VisitLocalVariableDeclaration(JavaParser.LocalVariableDeclaration
                 .Select(varDec => (AssignmentExpression)Visit(varDec))
                 .Where(initializer => initializer != null).ToArray();
 
+            if (initializers.Count() == 1 && initializers.First().Right is MultichildExpression multichildExpression)
+            {
+                var expressions = multichildExpression.Expressions;
+                // is array?
+                if (Helper.TryCheckIdTokenValue(expressions.First(), "{") &&
+                    Helper.TryCheckIdTokenValue(expressions.Last(), "}"))
```

Comment:
Do `expressions` always have at least one element? Add an additional check otherwise or use `FirstOrDefault` and `LastOrDefault` methods.